docs/cli-help.txt

```diff
@@ -22,14 +22,11 @@ optional arguments:
   -v, --verbose         This flag can be used multiple times to incrementally
                         increase the verbosity of output, e.g. -vvv. (default:
                         -2)
-  -t, --text            Use the text output instead of the curses UI.
```

Comment:
Yeah I think there are a number of things we could do to make this a little nicer for developers. I think it could get a bit tricky though.

And no need to make a separate PR. These changes happen automatically in our [release script](https://github.com/certbot/certbot/blob/master/tools/release.sh#L149).